src/Angor/Client/Pages/Spend.razor

```diff
@@ -87,34 +87,41 @@
     }
     else
     {
-        @if (totalInvestedAmount <= founderProject.ProjectInfo.TargetAmount && founderProject.ProjectHasStarted())
+        @if (Money.Satoshis(totalInvestedAmount).ToUnit(MoneyUnit.BTC) <= Money.Satoshis(founderProject.ProjectInfo.TargetAmount).ToUnit(MoneyUnit.BTC) && founderProject.ProjectHasStarted())
```

Comment:
why convert to btc unit? this we do only for display

Reply:
Yes, it is not needed here, it is only needed in the display.